index.js

```diff
@@ -41,78 +41,105 @@ module.exports = {
 
     return this._findHost();
   },
+  _debug() {
```

Comment:
remove?

Reply:
No on the contrary, keeping the hook visible so that ppl can uncomment the console.log when needed: best way to debug tree-shaking which I expect will be needed on a regular basis.